### PR TITLE
Adds a longer delay for server responses in integration tests

### PR DESCRIPTION
**Pull request by rcorral**

### CODE REVIEW

File: tests/_support/constants/Timeouts.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const Timeouts = {
+  SERVER_RESPONSE_DELAY: 200
```
Comment (mlunoe) [inline tests/_support/constants/Timeouts.js]: Let's get a comment here:
`// 200 seems to be enough time to make an assessment before the server responds`

**Comment by mlunoe** [comment]
Looks good man, let's add that comment and let's get this in!

**Comment by mlunoe** [comment]
@rcorral Agreed to try this out for ~10 consecutive runs to see if it passes. If that is the case we should be good to go!

**Comment by rcorral** [comment]
run integration tests

**Comment by rcorral** [comment]
Ran the integration tests locally 15 consecutive times and there were no failures.

**Comment by mlunoe** [comment]
🚢

**Comment by jfurrow** [comment]
I'm gonna do it

**Comment by jfurrow** [comment]
NOOOOO

**Pull request merged**